superbench/benchmarks/micro_benchmarks/__init__.py

```diff
@@ -6,5 +6,8 @@
 from superbench.benchmarks.micro_benchmarks.micro_base import MicroBenchmark, MicroBenchmarkWithInvoke
```

Comment:
Why include in this PR?

Reply:
this is due to merge guoshuai's pr, after merging the lastest main, it disappeared